File: 3024. Type of Triangle/Solution.py
class Solution:
    def triangleType(self, nums: list[int]) -> str:
        nums.sort()
        a, b, c = nums
        if a + b <= c:
            return "none"
        elif a == b == c:
            return "equilateral"
        elif a == b or b == c:
            return "isosceles"
        else:
            return "scalene"

File: 3024. Type of Triangle/Solution.cpp
#include <algorithm>
#include <ranges>
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
    string triangleType(vector<int>& nums)
    {
        ranges::sort(nums);
        if (nums[0] + nums[1] <= nums[2])
            return "none";
        else if (nums[0] == nums[1] && nums[0] == nums[2])
            return "equilateral";
        else if (nums[0] == nums[1] || nums[1] == nums[2])
            return "isosceles";
        else
            return "scalene";
    }
};
